plugins/custom_plugin.go

```diff
@@ -192,7 +198,17 @@ func (cp *CustomPlugin) processTar(fs afero.Fs, path string, targetDir string) e
 			return errs.NewUser("Nil tar file header")
 		}
 		// the target location where the dir/file should be created
-		target := filepath.Join(targetDir, header.Name)
+		splitTarget := strings.Split(header.Name, string(os.PathSeparator))
+		fmt.Println(splitTarget)
```

Comment:
this should probably be log.Debug

Reply:
@ryanking forgot to rm them, were just here for dev checks